boards/seeedstudio-xiao-esp32c3/Makefile.dep

```diff
@@ -1,9 +1,18 @@
-ifeq (,$(filter stdio_% slipdev_stdio,$(USEMODULE)))
-  USEMODULE += stdio_usb_serial_jtag
-endif
+ifneq (,$(filter periph_spi, $(USEMODULE)))
+  ifeq (,$(filter periph_init_buttons, $(DISABLE_MODULE)))
+    DISABLE_MODULE += periph_init_buttons
 
-ifneq (,$(filter saul_default,$(USEMODULE)))
+    MSG="Warning: Using periph_spi on Seeed Studio Xiao ESP32C3 board will disable BUTTON0 due to pin conflict."
+    $(shell $(COLOR_ECHO) "$(COLOR_RED)$(MSG)$(COLOR_RESET)" 1>&2)
```

Comment:
If it is only a warning, the message should be coloured yellow as usual, IMHO. It is confusing when you get the red message and the compilation works afterwards.